feat: add new test for increased coverage with keploy

Sonichigo opened the pull request:
This pull request introduces comprehensive unit tests for the `body-parser.js` and `error-handler.js` module, significantly improving the project's test coverage from `0%` to `80%+`. I have also added the new tests for `cookies.js` increasing from `66.66%` to `100%`

The new tests cover various scenarios, ensuring the reliability and robustness of the functionalities.

### Current Coverage
![image](https://github.com/user-attachments/assets/7e518475-7175-47bd-9af6-f7d63279c121)


### New Coverage
<img width="675" alt="image" src="https://github.com/user-attachments/assets/eeae300f-3058-479d-919a-8da0ca6c8bbb" />

## Changes:

- Developed unit tests targeting key functions .
- Validated edge cases and error handling mechanisms.
- Ensured compatibility with existing codebase and testing frameworks.

## Testing:

- All new and existing tests have been executed successfully.
- No regressions or issues were identified during testing.

## Impact:
This enhancement bolsters code reliability and facilitates future development by providing a more robust testing framework.

## Checklist:
- [x]  Code builds cleanly without errors or warnings.
- [x]  Unit tests have been added and pass successfully.

The PR content was generated automatically using [testing-agent](https://github.com/keploy/keploy)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Kong/insomnia-mockbin?pullRequest=182) <br/>All committers have signed the CLA.

Sonichigo commented:
recheck

jackkav commented:
If there's only white space change in package jsons please revert them.

Sonichigo commented:
hi @jackkav, i have revert the `package.json` and added required libraries, as whitespaces were introduced due to my linter extension.

jackkav commented:
> hi @jackkav, i have revert the `package.json` and added required libraries, as whitespaces were introduced due to my linter extension. 

We use biome to lint and format

Sonichigo commented:
> > hi @jackkav, i have revert the `package.json` and added required libraries, as whitespaces were introduced due to my linter extension.
> 
> We use biome to lint and format

Done, I have fixed linting and formatting via Biome, now. Thanks for pointing to biome.

Sonichigo commented:
Hi @jackkav any updates on the pr or it's contents?

jackkav commented:
I need to discuss further with the team.
